config/v1/types.go

```diff
@@ -5,19 +5,25 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
-// ConfigMapReference references a configmap in the openshift-config namespace.
-type ConfigMapReference struct {
+// ConfigMapFileReference references a configmap in the openshift-config namespace.
+type ConfigMapFileReference struct {
```

Comment:
similar to ConfigMapNameReference, I think this struct should denote the namespace expected for use where it is consumed in BuildConfig.